docs/.link-check-errors.txt

```diff
@@ -1,18 +1,5 @@
 Broken links found by `yarn docs:link-check` that exist:
 
-- https://mui.com/base-ui/react-autocomplete/hooks-api/#use-autocomplete
-- https://mui.com/base-ui/react-portal/components-api/
-- https://mui.com/material-ui/api/button/#props
-- https://mui.com/material-ui/api/input-base/#InputBase-prop-disableInjectingGlobalStyles
-- https://mui.com/material-ui/api/menu-item/#css
-- https://mui.com/material-ui/api/popper/#props
 - https://mui.com/material-ui/customization/right-to-left/
-- https://mui.com/material-ui/guides/api/#native-properties
+- https://mui.com/material-ui/integrations/styled-components/
```

Comment:
I've bumped `monorepo` locally and noticed that the existing link using `guides` has been replaced with `integrations`.
TL/DR.: This is a pointless addition due to the out-of-sync `monorepo`. 🙈